src/screens/SubTags/SubTags.tsx

```diff
@@ -0,0 +1,587 @@
+import { useMutation, useQuery, type ApolloError } from '@apollo/client';
+import { Search, WarningAmberRounded } from '@mui/icons-material';
+import SortIcon from '@mui/icons-material/Sort';
+import Loader from 'components/Loader/Loader';
+import IconComponent from 'components/IconComponent/IconComponent';
+import { useNavigate, useParams, Link } from 'react-router-dom';
+import type { ChangeEvent } from 'react';
+import React, { useState } from 'react';
+import { Form } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Modal from 'react-bootstrap/Modal';
+import Row from 'react-bootstrap/Row';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+import type { InterfaceQueryUserTagChildTags } from 'utils/interfaces';
+import styles from './SubTags.module.css';
+import { DataGrid } from '@mui/x-data-grid';
+import type { GridCellParams, GridColDef } from '@mui/x-data-grid';
+import { Stack } from '@mui/material';
+import {
+  CREATE_USER_TAG,
+  REMOVE_USER_TAG,
+} from 'GraphQl/Mutations/TagMutations';
+import {
+  USER_TAG_ANCESTORS,
+  USER_TAG_SUB_TAGS,
+} from 'GraphQl/Queries/userTagQueries';
+
+const dataGridStyle = {
+  '&.MuiDataGrid-root .MuiDataGrid-cell:focus-within': {
+    outline: 'none !important',
+  },
+  '&.MuiDataGrid-root .MuiDataGrid-columnHeader:focus-within': {
+    outline: 'none',
+  },
+  '& .MuiDataGrid-row:hover': {
+    backgroundColor: 'transparent',
+  },
+  '& .MuiDataGrid-row.Mui-hovered': {
+    backgroundColor: 'transparent',
+  },
+  '& .MuiDataGrid-root': {
+    borderRadius: '0.1rem',
+  },
+  '& .MuiDataGrid-main': {
+    borderRadius: '0.1rem',
+  },
+};
+
+function SubTags(): JSX.Element {
+  const { t } = useTranslation('translation', {
+    keyPrefix: 'organizationTags',
+  });
+  const { t: tCommon } = useTranslation('common');
+
+  const [addSubTagModalIsOpen, setAddSubTagModalIsOpen] = useState(false);
+
+  const { orgId: currentUrl, tagId: parentTagId } = useParams();
```

Comment:
Same about this `currentUrl`, this needs to be changed to `orgId` or `organizationId`